frontends/ol-util/src/components/PageBanner.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react"
```

Comment:
This is moved from `open-discussions`, verbatim, other than the simple flow -> ts conversion